solver/pb/ops.proto

```diff
@@ -71,7 +71,13 @@ message BuildInput {
 // OpMetadata is a per-vertex metadata entry, which can be defined for arbitrary Op vertex by both "script" and build client (e.g. buildctl).
 message OpMetadata {
 	bool ignore_cache = 1;
-	// TODO: add worker constraint, etc.
+	WorkerConstraint worker_constraint = 2;
+}
+
+// WorkerConstraint is WIP. Will be changed.
+message WorkerConstraint {
+	string worker_name = 1; // e.g. "runc-overlay", "containerd-overlay", "containerd-btrfs"
```

Comment:
I know this is WIP so maybe already planned, but I don't think worker name should be in here. We could have a `worker_type/worker_brand` (in addition to arch, os, labels etc). If you have 20 workers they would all have different names and adding/removing nodes would change them so this is not something that the client should know.